tests/manage/pv_services/test_verify_rwo_using_dc_pod.py

```diff
@@ -75,40 +73,50 @@ def wait_for_pods_and_verify(self):
         """
         # Wait for pods
         for pods in TimeoutSampler(
-            180, 2, func=pod.get_all_pods, namespace=self.namespace,
+            360, 2, func=pod.get_all_pods, namespace=self.namespace,
             selector=[self.name], selector_label='name'
         ):
-            if len(pods) == 2:
+            if len(pods) == 5:
```

Comment:
if meaning of this condition is `if len(pods) == replica_count` that defined in another function, plz consider using constant instead

Reply:
Done